ord_schema/validations_test.py

```diff
@@ -104,6 +104,31 @@ def test_orcid_should_fail(self):
         with self.assertRaisesRegex(validations.ValidationError, 'Invalid'):
             self._run_validation(message)
 
+    def test_synthesized_compound(self):
+        message = reaction_pb2.CompoundPreparation(
+            type='SYNTHESIZED', reaction_id='dummy_reaction_id')
+        self.assertEmpty(self._run_validation(message))
+        message = reaction_pb2.CompoundPreparation(
+            type='NONE', reaction_id='dummy_reaction_id')
+        with self.assertRaisesRegex(validations.ValidationError, 'IDs'):
```

Comment:
Maybe be a bit more descriptive with this regex? I'm not sure what the error is from reading just this.